infrastructure/transform/terragrunt.hcl

```diff
@@ -61,4 +61,6 @@ inputs = {
   core_kv_id            = dependency.core.outputs.core_kv_id
   core_kv_uri           = dependency.core.outputs.core_kv_uri
   subnet_address_spaces = dependency.core.outputs.subnet_address_spaces
+  local_mode            = dependency.core.outputs.local_mode
```

Comment:
You could remove this if added in the top-level terragrunt inputs instead

Reply:
Same with the deployer_ip_address as you've already added that to the inputs anyway

Reply:
ok i see, getting used to the dark arts of terragrunt...